src/index.js

```diff
@@ -150,31 +160,31 @@ module.exports = {
 
   sourceDirsFor(strategy) {
     return this.optionFor(strategy, 'sourceDirs')
-      .filter((sourceDir) => fs.existsSync(sourceDir));
+      .filter((sourceDir) => typeof sourceDir !== 'string' || fs.existsSync(sourceDir));
   },
 
-  originalSvgsFor: _.memoize(function(strategy) {
```

Comment:
This doesn't work correctly for instance methods -- it doesn't take `this` into account, so all instances of the class will share the same memoized value. In our case, this class will get instantiated once for each addon that depends on it, plus possibly once for the containing application (or dummy application).